libbeat/processors/add_cloud_metadata/provider_aws_ec2.go

```diff
@@ -0,0 +1,24 @@
+package actions
```

Comment:
At one point there was an actions folder and I think we decided in the original PR for this processor to move it into its own folder. Looks like I did the file move, but never updated the actual package name.